### DISCUSSION TITLE
Collaborating on ASReview?

### DISCUSSION THREAD

**Discussion by jessyoung828** [asreview/asreview#1713] [Q&A]
I see that there is a suggestion to have both team members upload the same data set and use the same articles to train the project. But we are doing a very loose systematic review and don't need to double code each and every article. Would it be a bad idea to individually work on two separate datasets but use the same training articles? 

For example, I upload a dataset of the first 5,000 (1 - 5,000) records and my partner uploads a dataset of the next 5,000 records (5,001-10,000). For the prior knowledge, we would both upload the same agreed upon 5 additional records that we know are relevant. Would this work? 

Essentially, I'd love to do option 3 on this site: https://asreview.nl/blog/seven-ways-to-integrate-asreview/#multiple but simultaneously rather than trading off back and forth.

**Comment by Rensvandeschoot** [comment]
This seems like a very good idea!! As an additional step, you could double-check the exclusions from each other, similar to the procedure described in [10.1186/s13643-024-02472-w](https://doi.org/10.1186/s13643-024-02472-w), where we corrected Noisy Labels, or similar to the last step of the [SAFE procedure](https://doi.org/10.1186/s13643-024-02502-7).
